app/exports/_common_columns.yml

```diff
@@ -2,7 +2,7 @@ extract_date:
   type: string
   format: date-time
   description: Time when the report ran
-  example: 2019-06-13T10:44:31Z
+  example: 2020-11-01T00:00:00+00:00
```

Comment:
This is how ISO 8601 strings are rendered in the exports, so I thought it worth updating these.